docs/ru/sql-reference/statements/create/table.md

```diff
@@ -243,8 +243,15 @@ CREATE TABLE codec_example
     timestamp DateTime CODEC(DoubleDelta),
     slow_values Float32 CODEC(Gorilla)
 )
-ENGINE = MergeTree()
+ENGINE = MergeTree();
 ```
+### Экспериментальные кодеки {#experimental-codecs}
```

Comment:
But we already removed them (the experiment has failed).
Now the setting remains with the following doc:

`If it is set to true, allow to specify experimental compression codecs (but we don't have those yet and this option does nothing).`

Reply:
@alexey-milovidov So the PR should be closed then. Pity.